com.github.nrittsti.NTag.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.nrittsti.NTag
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: ntag
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+modules:
+  - name: ntag
+    buildsystem: simple
+    build-commands:      
+      - install -dm755 /app/opt/ntag/lib
+      - install -Dm644 com.github.nrittsti.NTag.desktop /app/share/applications/com.github.nrittsti.NTag.desktop
+      - install -Dm644 com.github.nrittsti.NTag.metainfo.xml /app/share/metainfo/com.github.nrittsti.NTag.metainfo.xml
+      - install -Dm644 ntag.png /app/share/icons/hicolor/128x128/apps/com.github.nrittsti.NTag.png
+      - install -Dm755 com.github.nrittsti.NTag.sh /app/bin/ntag      
+      - install -Dm644 lib/*.jar /app/opt/ntag/lib
+      - install -Dm644 *.{jar,png,properties} /app/opt/ntag      
+      - install -dm755 /app/opt/ntag/jre
+      - cp -r --no-preserve=ownership --preserve=mode jre/* /app/opt/ntag/jre/
+    sources:
+      - type: archive
+        url: https://github.com/nrittsti/ntag/releases/download/v1.2.8/ntag-1.2.8-linux_bin.tar.gz
```

Comment:
Any reason to make it specific to x86_64? If you base it on JDK extension, it would likely work on aarch64 too. For example: https://github.com/flathub/com.diy_fever.DIYLayoutCreator

Reply:
NTag depends on JavaFX (OpenJFX) as GUI toolkit. This library (.jar) is unfortunately platform dependent and contains compiled .so files for x86_64. As far as I know there is no ARM64 Version available. If i am wrong please let me know. The JDK extension sounds good, i will take a closer look.

Reply:
https://repo.maven.apache.org/maven2/org/openjfx/javafx-graphics/15.0.1/

Reply:
after some experiments I have found a setup that works:
NTag / Java 15 build in combination with
Gnome and OpenJDK extension